core/services/feeds/service.go

```diff
@@ -43,6 +43,22 @@ var (
 		Name: "feeds_job_proposal_requests",
 		Help: "Metric to track job proposal requests",
 	})
+	promJobProposalsPending = promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
Would a Gauge Vector be a better structure to use here? 

https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#GaugeVec

`Help:      "Number of job proposals partitioned by status.",`

Reply:
Yea will allow us to drop three variables.